packages/webpack-plugin/README.md

```diff
@@ -52,7 +52,7 @@ const config = (env, argv) => ({
       classNamePrefix: 'x',
       // Required for CSS variable support
       unstable_moduleResolution: {
-        // type: 'commonJS' | 'haste'
+        // type: 'commonJS' | 'esm' | 'haste'
```

Comment:
My intention was to simply try both `require.resolve` *and* `import.resolve` instead of just `require.resolve`. That wouldn't need an extra option for configuration?

What do you think about that? Themes are essentially ESM only anyway.